Userland/Libraries/LibJS/Tests/builtins/String/String.prototype.normalize.js

```diff
@@ -0,0 +1,39 @@
+test("Function length is 0", () => {
+    expect(String.prototype.normalize).toHaveLength(0);
+});
+
+test("Function name is normalize", () => {
+    expect(String.prototype.normalize.name).toBe("normalize");
+});
+
+test("Type is function", () => {
+    expect(typeof String.prototype.normalize).toBe("function");
+});
+
+test("Invalid form throws", () => {
+    expect(() => "foo".normalize("bar")).toThrow(RangeError);
+    expect(() => "foo".normalize("NFC1")).toThrow(RangeError);
+    expect(() => "foo".normalize(null)).toThrow(RangeError);
```

Comment:
Please prefer `toThrowWithMessage` with the full error message to further harden against unintended regressions